docker-compose.yml

```diff
@@ -0,0 +1,25 @@
+version: '3'
+
+services:
+  inbound:
+    image: local/inbound
+    ports:
+      - "443:443"
+    environment:
+      - MHS_LOG_LEVEL=NOTSET
+  outbound:
+    image: local/outbound
+    ports:
+      - "80:80"
+    environment:
+      - MHS_LOG_LEVEL=NOTSET
+  dynamodb:
+    image: amazon/dynamodb-local
+    ports:
+      - "8000:8000"
+  rabbitmq:
+    build: ./common/local-env/rabbit/
+    ports:
+      - "15672:15672"
+      - "5672:5672"
+    hostname: "localhost"
```

Comment:
I've moved the running section to the top level `README.md`, and changed the wording a bit to describe starting a development cluster since as you say, it will also be used for the SCR adaptor.